addons/mass_mailing/views/snippets/s_comparisons.xml

```diff
@@ -28,25 +28,25 @@
                 <div class="col-lg-6 s_col_no_bgcolor pt32 pb16">
                     <div class="card o_cc o_cc3">
                         <div class="card-header"><h2 style="text-align: center;"><span style="font-weight: bolder;">PRO</span></h2></div>
-                        <div class="bg-white pt24 pb24">
+                        <div class="bg-white pt24 pb24" style="text-align: center;">
                             <ul class="list-group list-group-flush">
                                 <li class="list-group-item">
                                     <h3 class="card-title" style="text-align: center;">$18</h3>
                                     <p class="card-title" style="text-align: center;"><span style="font-size: 11px">user / month (billed annually)</span></p>
                                 </li>
                                 <li class="list-group-item">
-                                    <p style="text-align: center;"><span style="font-weight:bolder">Advanced</span>
+                                    <p><span style="font-weight:bolder">Advanced</span>
```

Comment:
Same here, with `fw-bolder` ?

Reply:
After discussion, since this is snippet is to be inserted and edited through the web editor, use of the style tag is preferred